flow/aggregates.py

```diff
@@ -549,23 +549,23 @@ def _register_aggregates(self, project):
         self._project = project
 
 
-def get_aggregate_id(jobs):
+def get_aggregate_id(aggregate):
     """Generate hashed id for an aggregate of jobs.
 
     Parameters
     ----------
-    jobs : tuple of :class:`~signac.contrib.job.Job`
-        The signac job handles.
+    aggregate : tuple of :class:`~signac.contrib.job.Job`
```

Comment:
Even if lists / other sequences would work, I would like the documentation to state that this _expects_ a `tuple`. Anecdotally, I haven't seen any instances in the code base where lists / non-tuples are used.

Reply:
The code definitely used to, but I haven't done a thorough review since the next branch was created and it's possible that we eventually cleaned up all the use case. As long as we've actually standardized that usage I'm fine with it, we definitely agreed that tuples are the _correct_ storage mechanism.